src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -74,8 +74,10 @@
     private boolean createFingerprint = true;
     private boolean skipTagLatest;
 
+    private String buildAdditionalArgs = "";
+
     @Deprecated
-    public DockerBuilder(String repoName, String repoTag, boolean skipPush, boolean noCache, boolean forcePull, boolean skipBuild, boolean skipDecorate, boolean skipTagLatest, String dockerfilePath) {
```

Comment:
could you keep the previous constructor for binary compatibility ? it should be enough with the @DataBoundSetter